lib/assigner/include/nil/blueprint/assigner.hpp

```diff
@@ -24,23 +19,17 @@ namespace nil {
 
             using ArithmetizationType = crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType>;
 
-            assigner(crypto3::zk::snark::plonk_table_description<BlueprintFieldType> desc,
-                     std::vector<assignment<ArithmetizationType>> &assignments) {
-                handler_ptr = std::make_shared<handler<BlueprintFieldType>>(desc, assignments);
+            assigner(std::vector<assignment<ArithmetizationType>> &assignments) {
```

Comment:
Seems like `assigner` class now is only a factory to create `handler`. Do we really need it? Maybe we could drop it or rename `handler` to `assigner`.

Reply:
yes, now `assigner` just factory for `handler`. But i suspect we may need introduce functionality depends on `crypto3` or `blueprint` here. For example: filling static part of assignment tables by `blueprint` interfaces. or process result tables (split for few provers, some actions in case of execute return error, ...)